### ISSUE TITLE
Best place to link to documentation for partial toolchains / single components

### ISSUE THREAD

**Issue by mjordan** [MarcusBarnes/mik#372]
The wiki has entries for complete toolchains, but we now have several components that are drop-in replacements for compontents documented as part of existing toolchains. The two components are the [new Templated metadata parser](https://github.com/MarcusBarnes/mik/issues/363) and the [Islandora OBJ OAI-PMH filegetter](https://github.com/MarcusBarnes/mik/issues/347). The former is intended for production while the latter is primarily intended for demonstration purposes (although it might prove useful in the future for migrations). 

We also have the "CSV to JSON" toolchain, which is documented on the [Information for Developers](https://github.com/MarcusBarnes/mik/wiki/Information-for-developers) page. Like the Islandora OBJ OAI-PMH filegetter, this toolchain in intended for demonstration/example purposes only.

My inclination is to add a section that is a sibling of "Toolchains", maybe calling it "Single components and demonstration toolchains". That would  cover the miscellaneous stuff but I fear would not be that clear or useful to most end users. Anyone have any other suggestions?

**Comment by MarcusBarnes** [comment]
Cookbook entries could be another possibility to consider.

**Comment by mjordan** [comment]
@MarcusBarnes yes that is a good idea, and it would be simple to incorporate links to the cookbook entries from the existing toolchains' wiki entries (see the second paragraph of my original issue, above).

**Comment by mjordan** [comment]
OK, links to the Cookbook entry on the Templated Metadata Parser and accompanying manipulator are in my latest comment in #373.

**Comment by mjordan** [comment]
Ha. Too many issues. I just #358 takes care of the Islandora OBJ OAI-PMH Filegetter.
